src/utils/file_ops.cpp

```diff
@@ -28,16 +28,31 @@ bool mp::FileOps::isReadable(QDir& dir)
     return dir.isReadable();
 }
 
+bool mp::FileOps::mkpath(QDir& dir, const QString& dirName)
+{
+    return dir.mkpath(dirName);
+}
+
 bool mp::FileOps::rmdir(QDir& dir, const QString& dirName)
 {
     return dir.rmdir(dirName);
 }
 
+bool mp::FileOps::exists(QFile& file)
```

Comment:
Should be able to do this:

```suggestion
bool mp::FileOps::exists(const QFile& file)
```